common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -159,6 +159,7 @@ class @Problem
         @setupInputTypes()
         @bind()
         @queueing()
+        @renderProgressState()
```

Comment:
@ormsbee, I think, we are not expecting any response in the result of some AJAX request like we're expecting in `else` part. So, in `if` part, I am only calling `renderProgressState` to render status on the basis of what's being set in the data-attributes (which have already been set during template rendering). User clicking check button is really the case when need to kick of AJAX and get response to update these `progress_status` & `progress_details` data-attributes in order for `renderProgressState` to render updated progress status. Also on reload, template rendering will occur again and we will get update values in these data-attributes.